_docs/services/relational-database.md

```diff
@@ -116,13 +116,11 @@ Name               | Description
 A couple of notes regarding the optional `version` parameter:
 
 - It is currently only supported for dedicated MySQL and PostgreSQL instances; if you specify it for any other type of instance it is ignored.
-- It only supports major version numbers (e.g. "8.0"); if you specify a minor/patch level version (e.g., "11.8" for PostgreSQL or "8.0.32" for MySQL), the command will fail.
+- It only supports major version numbers (e.g. "8.0"); if you specify a minor/patch level version (e.g., "12.8" for PostgreSQL or "8.0.32" for MySQL), the command will fail.
```

Comment:
```suggestion
- It only supports major version numbers (e.g. "8.0"); if you specify a minor/patch level version (e.g., "12.16" for PostgreSQL or "8.0.32" for MySQL), the command will fail.
```

12.8 is deprecated: https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-versions.html#postgresql-versions-version12